09-gis.Rmd

```diff
@@ -461,7 +462,7 @@ library(link2GI)
 link = findGRASS() 
 ```
 
-`link` is a `data.frame` which contains in its rows the GRASS 7 installations on your computer. 
+`link` is a `data.frame` which contains in its rows the GRASS installations on your computer. 
```

Comment:
Ok, I've just looked at the source code of `link2GI::findGRASS()`, and in fact, it looks for `grass??` which very likely means that also other older or newer GRASS executables would be found. Thanks for spotting!